test/util/edu-helpers.js

```diff
@@ -16,6 +18,35 @@ function completeVeteranInformation(client, data, onlyRequiredFields) {
   }
 }
 
+function completeRelativeInformation(client, data, onlyRequiredFields) {
+  const dobFields = data.relativeDateOfBirth.split('-');
+  client
+    .clearValue('input[name="root_relativeFullName_first"]')
+    .setValue('input[name="root_relativeFullName_first"]', data.relativeFullName.first)
```

Comment:
Perhaps this is unnecessary, but would it be valuable to create a function that both clears and sets the value, taking as arguments the field and the new value? Only benefits I suppose would be that it reduce all these lines by half and would ensure you're always clearing the field before trying to set the value?